WasatchNET/SPISpectrometer.cs

```diff
@@ -477,7 +898,52 @@ public override ushort secondaryADC
 
         public override string firmwareRevision => "";
 
-        public override string fpgaRevision => "";
+        public override string fpgaRevision
+        {
+            get
+            {
+                byte[] payload = new byte[0];
+
+                byte[] command = wrapCommand(GET_FPGA_REV, payload, 40);
```

Comment:
If 40 bytes is going to be a standard padding, that would make a good constant too.